gulpfile.js

```diff
@@ -58,27 +155,64 @@ gulp.task('css:lint', () => {
         }));
 });
 
-gulp.task('static:clean', () => {
-    return del(['static/**', '!static', '!static/.gitignore']);
+gulp.task('clean', () => {
+    return del(['static_build']);
 });
 
-gulp.task('default', () => {
-    gulp.start('media:watch');
+gulp.task('assets', () => {
+    return gulp.src([
+        'media/**/*',
+        'node_modules/@mozilla-protocol/core/**/*',
+        '!node_modules/@mozilla-protocol/core/*'
+    ]).pipe(gulp.dest('static_build'));
+});
 
-    gulp.watch(lintPathsJS).on('change', file => {
-        return gulp.src(file.path)
-            .pipe(eslint())
-            .pipe(eslint.format());
+gulp.task('browser-sync', ['js:compile', 'css:compile'], () => {
+    var proxyURL = process.env.BS_PROXY_URL || 'localhost:8000';
+    var openBrowser = !(process.env.BS_OPEN_BROWSER === 'false');
+    browserSync({
+        proxy: proxyURL,
+        open: openBrowser,
+        serveStatic: [{
+            route: '/media',
+            dir: 'static_build'
+        }]
     });
+});
 
-    gulp.watch(lintPathsCSS).on('change', file => {
-        return gulp.src(file.path)
-            .pipe(gulpStylelint({
-                failAfterError: false,
-                reporters: [{
-                    formatter: 'string',
-                    console: true
-                }]
-            }));
-    });
+gulp.task('reload-other', ['assets'], browserSync.reload);
+gulp.task('reload-css', ['css:compile'], browserSync.reload);
+gulp.task('reload-js', ['js:compile'], browserSync.reload);
+gulp.task('reload', browserSync.reload);
+
+// --------------------------
+// DEV/WATCH TASK
+// --------------------------
+gulp.task('watch', ['browser-sync'], function () {
```

Comment:
Nit: this function and a few others can turned into shorthand arrow functions `=>`